### PR TITLE
Fix USRBG

### PR DESCRIPTION
**Pull request by AutumnVN**
:3

### CODE REVIEW

**Comment by Nuckyz** [comment]
You will not do any temporary fix, also bad bad stuff modying the premium type

File: src/plugins/usrbg/index.tsx [hunk 1]
```diff
@@ -51,6 +51,13 @@ export default definePlugin({
                 match: /(\i)\.bannerSrc,/,
                 replace: "$self.useBannerHook($1),"
             }
+        },
+        {
+            find: "R.ZP.isPremiumAtLeast",
```
Comment (Vendicated) [inline src/plugins/usrbg/index.tsx:56]: erm

File: src/plugins/usrbg/index.tsx [hunk 2]
```diff
@@ -51,6 +51,13 @@ export default definePlugin({
                 match: /(\i)\.bannerSrc,/,
                 replace: "$self.useBannerHook($1),"
             }
+        },
+        {
+            find: ".displayProfile,",
+            replacement: {
+                match: /(\i)(\.premiumType)/,
+                replace: "$self.premiumHook($1) || $1$2"
```
Comment (Vendicated) [inline src/plugins/usrbg/index.tsx:59]:
```suggestion
                replace: "$self.premiumHook($1) || $&"
```

**Comment by Vendicated** [comment]
ur original find found 5 different modules. I recommend u use the Vencord Companion extension to test ur patches!!

**Review by Vendicated — APPROVED** [review]

**Pull request merged**